server/sonar-server/src/main/java/org/sonar/server/component/ws/ComponentsWs.java

```diff
@@ -39,7 +39,7 @@ public ComponentsWs(AppAction appAction, SearchViewComponentsAction searchViewCo
   public void define(Context context) {
     NewController controller = context.createController("api/components")
       .setSince("4.2")
-      .setDescription("Components management");
+      .setDescription("Get information about a component (file, directory, project, ...) and its ancestors or children.");
```

Comment:
I don't agree with the provided description if we take into account internal WS ex: api/components/search